src/codegeneration/SymbolTransformer.js

```diff
@@ -80,6 +82,8 @@ function isSafeTypeofString(tree) {
   return true;
 }
 
+var runtimeOption = parseStatement `$traceurRuntime.options.symbols = true`;
```

Comment:
The aim is to emit 
```js
$traceurRuntime.options.symbols = true;
```
iff the `SymbolTransform` is applied.  All JS that can see the same global will see the same value.